nodejs/v10-community/rest-http/booster.yaml

```diff
@@ -4,10 +4,6 @@ source:
   git:
     url: https://github.com/nodeshift-starters/nodejs-rest-http
     ref: master
```

Comment:
This change belongs to the same PR?

Reply:
Needs cleaning up @tinakurian  :)

Reply:
Sorry, i wasn't able to see Go show up as an option without disabling this first. Will undo this. Thanks!